### ISSUE TITLE
Add a env variable to make latest a hard copy

### ISSUE THREAD

**Issue by fredericoschardong** [prodrigestivill/docker-postgres-backup-local#151]
Symlinks are interesting to save space. However, I would like to have a volume in my postgress container to restore the latest backup in case I have to remove the postgres container and its volume for some reason.  E.g.:

```
  db:
    image: bitnami/postgresql:14.11.0
    restart: always
    volumes:
      - BACKUP_DIR/last/DB-latest.sql.gz: /docker-entrypoint-initdb.d/backup.sql.gz:ro
```

Can we have a env variable to ensure `latest.sql.gz` is a hard copy instead of a symlink?

**Comment by prodrigestivill** [comment]
not sure it would work with hard link for what I know about container sharing.
have you tested that changing the hardlink after the container is launched is propagated inside the containers?

What are your concerns in sharing the whole volume with the restore service?

**Comment by fredericoschardong** [comment]
I could share the volume. However, the issue is that postgres wouldn't know which file to restore. My guess is that it would try to load them all. If latest were a regular file, I could only mount it.

Is there another way around this?

**Comment by prodrigestivill** [comment]
Implemented in 4d1f8604f62a4428b55f61f9ccee786c31eac335.

Again, as said, I believe that this will only work if the container intended to have this file mapped gets restarted.
So I don't believe you will not get this file updated automatically during the execution of the other container (by limitation of the docker engine), but it will work if you restart it.

You need to set:
```yaml
env:
   BACKUP_LATEST_TYPE: hardlink
```
